packages/recs/package.json

```diff
@@ -22,6 +22,9 @@
     "release": "npm publish --access=public"
   },
   "devDependencies": {
+    "@latticexyz/cli": "^1.41.0",
+    "@latticexyz/schema-type": "^1.41.0",
+    "@latticexyz/std-client": "^1.41.0",
```

Comment:
`recs` is a requirement of `std-client` too, so this creates a circular dependency. Is there a way we can avoid `recs` depending on `std-client`?